src/gui/ui_settings.c

```diff
@@ -442,16 +444,17 @@ static int settings_loop(int id, void *context, const input_data *input) {
       if (!left && !right) {
           break;
       }
-      char *resolutions[] = {"960x540", "960x544", "1280x720", "1920x1080"};
+      char *resolutions[] = {"960x544", "960x540", "1280x720", "1920x1080", "1280x540"};
       sprintf(current, "%dx%d", config.stream.width, config.stream.height);
 
-      new_idx = move_idx_in_array(resolutions, 4, current, left ? -1 : +1);
+      new_idx = move_idx_in_array(resolutions, 7, current, left ? -1 : +1);
```

Comment:
it could be 5 isn't it?

Reply:
Yeah this is a leftover when I removed some resolutions that are not supported by Vita 1000. Do you know why we keep 1920x1080 in the list? This is not supported by Vita 1000 as well.

Reply:
Fixed in https://github.com/xyzz/vita-moonlight/pull/169/commits/cb3652913c777ddda088ab162b3a85fd57222b32

Reply:
this resolution also has meaning of the remove computer's resolution.
historical reason, most games require generic resolution

Reply:
Then I'll leave this as is for now. I'm using desktop streaming.